modules/log-collection-rbac-permissions.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/log-forwarding.adoc
+
+:_content-type: PROCEDURE
+[id="log-collection-rbac-permissions_{context}"]
+= Authorizing log collection RBAC permissions
+
+In logging 5.8 and newer versions, the Cluster Logging Operator provides `collect-audit-logs`, `collect-application-logs`, and `collect-infrastructure-logs` cluster roles, which correspond to roles that are able to collect audit logs, application logs, and infrastructure logs respectively.
+
+You can authorize RBAC permissions for log collection by binding the required cluster roles to a service account.
+
+.Prerequisites
+
+* The Cluster Logging Operator is installed in the `openshift-logging` namespace.
+* You have administrator permissions.
+
+.Procedure
+
+. Create a service account for the collector. If you want to write logs to storage that requires a token for authentication, you must include a token in the service account.
+
+. Bind the appropriate cluster roles to the service account:
++
+.Example binding command
+[source,terminal]
+----
+$ oc create clusterrolebinding <cluster_role_binding_name> --clusterrole=<cluster_role_name> --serviceaccount=<namespace:service_account_name>
```

Comment:
Not sure which command is better?
oc adm policy add-cluster-role-to-user  <cluster_role_name> - system:serviceaccount:<namespace>:<ervice_account_name>

Reply:
I'm indifferent here but @anpingli example lends itself more to an "admin is doing something" and looks more "admin" like.  I suspect you will need elevated permissions for both :man_shrugging: